src/tests/inpty.cc

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
I decided to take this, deciding better was the enemy of good enough.

I also tried mosh-1.2.2 on Cygwin 2.6.0 (not quite current) and it worked fine.  What other pain was `IO::Pty` causing?